cibuildwheel/windows.py

```diff
@@ -107,8 +158,12 @@ def shell(args, env=None, cwd=None):
         shell(['python', '-c', '"import struct; print(struct.calcsize(\'P\') * 8)\"'], env=env)
 
         # prepare the Python environment
-        shell(['python', '-m', 'pip', 'install', '--upgrade', 'pip'],
+        if config.version == "3.4.x":
+            shell(['python', '-m', 'pip', 'install', 'pip==19.1.1'],
               env=env)
+        else:
+            shell(['python', '-m', 'pip', 'install', '--upgrade', 'pip'],
+                env=env)
```

Comment:
Is this change still needed?

Reply:
This is not connected with travis but generally with python. From version 19.2 `pip` drop support of python 3.4. and I see errors in test when calling upgrade. on python 3.4

Reply:
I see, thanks